crates/solidity/inputs/language/src/lib.rs

```diff
@@ -15,6 +15,10 @@ pub use grammar::GrammarConstructorDslV2;
 use anyhow::Result;
 use codegen_schema::types::{LanguageDefinition, LanguageDefinitionRef};
 
+thread_local! {
+ pub static LANGUAGE_DEF: codegen_language_definition::model::Language = SolidityDefinition::create();
```

Comment:
nit suggestion, as "language" is already included in the crate name:

```suggestion
 pub static DEFINITION: codegen_language_definition::model::Language = SolidityDefinition::create();
```

Reply:
Also, I wonder what is the case for caching this? since it is usually a single build script or a CLI that is creating/using it once.